tests/appsec/test_appsec_trace_utils.py

```diff
@@ -1,13 +1,27 @@
+import pytest
+
 from ddtrace import constants
 from ddtrace.appsec._constants import APPSEC
 from ddtrace.appsec.trace_utils import track_custom_event
 from ddtrace.appsec.trace_utils import track_user_login_failure_event
 from ddtrace.appsec.trace_utils import track_user_login_success_event
+from ddtrace.contrib.trace_utils import set_user
+from ddtrace.ext import SpanTypes
 from ddtrace.ext import user
+from ddtrace.internal import _context
+from tests.appsec.test_processor import RULES_GOOD_PATH
+from tests.appsec.test_processor import tracer_appsec  # noqa: F401
```

Comment:
The noqa and the other below are because flake8 seems to have problems understanding that this import is not redefined below (but it won't work without the import, of course).